### ISSUE TITLE
question on traning

### ISSUE THREAD

**Issue by xi-mao** [STVIR/pysot#70]
I use youtubebb and imageVID  to train siamrpn_alex_dwxcorr_16gpu。 I crop youtubebb size  255*255 and crop ImagenetVID  511*511。is it ok to crop like this？dose it effect the training process？   From my traning，  the loss declined well。Then  I use checkpoint_e50.pth to  test OTB100。and I found the tracker cann't track object at all。I don't know why 。is there something wrong with my crop ways？  My  cls loss finnally is 0.04 and my loc_loss finally is  0.21 from the tensorboard。

besides，dose the json file keep the object origin size or the scle size after crop？ I keep the origin size。

Can anyone explain my confusing？

**Comment by xi-mao** [comment]
I have solved the problem。 I think the project shoud add more description on train and test。 Before I mistake something

**Comment by beihaifusang** [comment]
Hi, sorry, I am facing a similar issue in preparing the cropped image for training. May I ask if it is okay to crop to, say 255x255, instead of the 511x511 as the training set? 

I would like to do so because it seems the dataset loader will crop & resize the loaded image to the correct size in the training process, but I am not sure if it's alright.

Thanks in advance.

**Comment by buaacarzp** [comment]
> 嗨，抱歉，在准备裁剪后的图像进行训练时，我面临类似的问题。请问是否可以裁剪为255x255，而不是511x511作为训练集？
> 
> 我想这样做是因为看来数据集加载器会在训练过程中将加载的图像裁剪并调整大小为正确的大小，但是我不确定是否还可以。
> 
> 提前致谢。

I find something that we should  crop the data in advance,I always think that it should be done  before feed the data into the net.

**Comment by sansanfree** [comment]
hello, i found the same problem with yours " From my traning， the loss declined well。Then I use checkpoint_e50.pth to test OTB100。and I found the tracker cann't track object at all", and my training dataset youtubebb is cropped with size 271*271, notdifferent with the size 511*511 like the project,  you said you have solved this problem , i would very thankful to get your reply
